src/Fixer/FunctionNotation/ReturnTypeDeclarationFixer.php

```diff
@@ -53,8 +103,31 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
     /**
      * {@inheritdoc}
      */
-    protected function getDescription()
+    public function getDefinition()
     {
-        return 'There should be no space before colon and one space after it in return type declaration.';
+        $versionSpecification = new VersionSpecification(70000);
+
+        return new FixerDefinition(
+            'There should be one or no space before colon, and one space after it in return type declarations, according to configuration.',
+            array(
+                new VersionSpecificCodeSample(
+                    "<?php\nfunction foo() : string {};",
+                    $versionSpecification
+                ),
+                new VersionSpecificCodeSample(
+                    "<?php\nfunction foo() : string {};",
+                    $versionSpecification,
+                    array('space_before' => 'none')
+                ),
+                new VersionSpecificCodeSample(
+                    "<?php\nfunction foo(): string {};",
+                    $versionSpecification,
+                    array('space_before' => 'one')
+                ),
+            ),
+            null,
```

Comment:
I would put sth like :  `'Rule is applied only on PHP 7+ environment.'` 
Could be anything, not insisting on exact content, just want to have required php printed ;)